### PR TITLE
WIP: Implements `times` builtin

### PR DESCRIPTION
**Pull request by timetoplatypus**
Implements #441

### CODE REVIEW

**Comment by timetoplatypus** [comment]
Not ready for merge yet, just creating the PR so progress can be tracked.

File: osh/builtin.py [hunk 1]
```diff
@@ -277,6 +280,28 @@ def _AppendParts(s, spans, max_results, join_next, parts):
...
 
 
+TIMES_SPEC = _Register('times')
+
+class Times(object):
+  def __call__(self, arg_vec):
+    if "SC_CLK_TCK" in os.sysconf_names:
+      clock_ticks_per_second = os.sysconf("SC_CLK_TCK")
+      # Convert resource usage (clock ticks) to seconds before pretty printing
+      print("%dm%1.3fs %dm%1.3fs" %
```
Comment (andychu) [inline osh/builtin.py]: Hm I don't know this API / builtin very well, but can it be done with os.times() ?

https://docs.python.org/2/library/os.html#os.times

Comment (timetoplatypus) [reply osh/builtin.py]: Ya, that looks like that'll work better. I'll test it out

**Comment by timetoplatypus** [comment]
Looks like `os.times()` works, and it's a good bit cleaner, too.

**Comment by andychu** [comment]
OK excellent!  Thanks for doing this.

Although this reminds me that we have no floating point in the stripped down CPython build !  

https://github.com/oilshell/oil/blob/master/native/libc.c#L358

However maybe this will still work with `mycpp` translation and we won't have to do that.  (Because `oil-native` is going to be the new Oil, as mentioned on the blog)

Either way this is great and gets us further.  Let me know on Zulip if you want to pick up any other tasks.

**Pull request merged**